product_tag_pricelist/models/product_pricelist_item.py

```diff
@@ -9,82 +9,93 @@ class PricelistItem(models.Model):
     _inherit = "product.pricelist.item"
 
     applied_on = fields.Selection(
-        selection_add=[("2a_product_tags", "Product Tags")],
+        selection_add=[
+            ("2a_product_tags", "Product Tags"),
+            ("2_product_category",),
```

Comment:
Why are you adding existing keys?

Reply:
hummmmmm the sistema not duplicate the keys?

Reply:
No

Reply:
It should be noted in the roadmap that adding new keys, etc., could alter the order. All of this must be taken into account, as it can lead to undesirable results with modules that also affect this.

Reply:
Done